test/xspec.bats

```diff
@@ -1434,6 +1458,33 @@ load bats-helper
     [ "${lines[17]}" = "passed: 4 / pending: 0 / failed: 0 / total: 4" ]
 }
 
+@test "Saxon 11 and later: invoking xspec using SAXON_HOME finds Saxon jar and XML Catalog Resolver jar" {
+    if [ -z "${BUNDLED_XML_RESOLVER}" ]; then
+        skip "Saxon 11 and later only"
+    fi
+    export SAXON_HOME="${work_dir}/saxon ${RANDOM}"
+    mkdir "${SAXON_HOME}"
+    mkdir "${SAXON_HOME}/lib"
+    cp "${SAXON_JAR}" "${SAXON_HOME}"
+    saxon_jar_dir="$(dirname -- "${SAXON_JAR}")"
+    ls "${saxon_jar_dir}"/lib
+    cp "${saxon_jar_dir}"/lib/xmlresolver* "${SAXON_HOME}"/lib/
+    cp "${saxon_jar_dir}"/lib/jline* "${SAXON_HOME}"/lib/
```

Comment:
Initially, I wanted the first piece of this command to be something like `cp "${saxon_jar_dir}/lib/*.jar" ...` without explicit names of the jar files. I got errors like "cp: cannot stat '/tmp/xspec-test-deps/saxon-11.5/lib/*.jar': No such file or directory". While troubleshooting, I inserted the `ls` command. After I found a syntax that worked, I left the `ls` command in place, thinking that the output might help if this test fails in the future.

Reply:
Then it would be nice if a comment indicating that the line with the `ls` command is specifically for troubleshooting purposes is added above it.

Reply:
Aside: The idea is that the maven artifact will be used in systems (Maven, Gradle, etc.) that manage dependencies. So the maven artifact depends on the resolver and jline. That should "just work" in Maven etc. We assume that users who just download the ZIP file from saxonica.com will want to be able to unzip it and run `java -jar saxon-xx.jar ...` so the required dependencies are bundled in `lib/` and the manifest points the classpath at those jars so that that "just works" too.

If there's a better approach here, I'm all ears.

Reply:
Fixed in 7f34f8b2b728ce250d5bf7c3d5ea19bd8d78de80. @AirQuick, thanks for the good idea. @ndw, thanks for your aside, which I had in mind when writing the comment in 0c0c96c2e1abb88e1754699af024bebd73cf0ff5.